src/app/system/system-routing.module.ts

```diff
@@ -59,10 +71,99 @@ const routes: Routes = [
       },
       {
           path: 'external-services',
-          component: ExternalServicesComponent,
```

Comment:
The commit should show only the removal of this line. Please remove any space or indentation change which makes it remove 4 lines instead.

Reply:
There are no space or indentation change in that, it is restructured that's why it showing that. All 4 lines are changed so it shows that they are removed.